src/xdr/Stellar-transaction.x

```diff
@@ -646,6 +666,27 @@ default:
     void;
 };
 
+/******* BumpSequence Result ********/
+
+enum BumpSequenceResultCode
+{
+    // codes considered as "success" for the operation
+    BUMP_SEQ_SUCCESS = 0,
+    // codes considered as "failure" for the operation
+    BUMP_SEQ_NOT_SUPPORTED_YET = -1, // The network hasn't moved to this protocol change yet
+    BUMP_SEQ_NO_ACCOUNT = -2,    // Trying to bump an account that doesn't exist
```

Comment:
opNO_ACCOUNT should take care of that

Reply:
See tests here: https://github.com/stellar/stellar-core/blob/master/src/transactions/MergeTests.cpp#L259

For reference how this works in different protocol versions.